components/features.tsx

```diff
@@ -1,322 +1,57 @@
-                  className="fill-current text-primary-500"
-                  width="64"
-                  height="64"
-                  rx="32"
-                />
-                <g
-                  transform="translate(22 21)"
-                  strokeLinecap="square"
-                  strokeWidth="2"
-                  fill="none"
-                  fillRule="evenodd"
-                >
-                  <path
-                    className="stroke-current text-white-main"
-                    d="M17 22v-6.3a8.97 8.97 0 003-6.569A9.1 9.1 0 0011.262 0 9 9 0 002 9v1l-2 5 2 1v4a2 2 0 002 2h4a5 5 0 005-5v-5"
-                  />
-                  <circle
-                    className="stroke-current text-white-main"
-                    cx="13"
-                    cy="9"
-                    r="3"
-                  />
-                </g>
-              </svg>
-              <h4 className="h4 mb-2 text-center">Never Start From Scratch</h4>
-              <p className="text-center text-lg text-gray-600">
-                Pear comes with high-quality templates and boilerplate code for
-                any type of project.
-              </p>
-            </div>
+  return (
+    <section className="flex flex-col justify-evenly">
+      {featureInfo.map((info, index) => (
+        <SingleFeature key={index} info={info}/>
+      ))}
+    </section>
+  )
+}
 
-            {/* 5th item */}
-            <div
-              className="relative flex flex-col items-center"
-              data-aos="fade-up"
-              data-aos-delay="400"
-              data-aos-anchor="[data-aos-id-blocks]"
-            >
-              <svg
-                className="mb-4 h-16 w-16"
-                viewBox="0 0 64 64"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <rect
-                  className="fill-current text-primary-500"
-                  width="64"
-                  height="64"
-                  rx="32"
-                />
-                <g
-                  strokeLinecap="square"
-                  strokeWidth="2"
-                  fill="none"
-                  fillRule="evenodd"
-                >
-                  <path
-                    className="stroke-current text-white-main"
-                    d="M29 42h10.229a2 2 0 001.912-1.412l2.769-9A2 2 0 0042 29h-7v-4c0-2.373-1.251-3.494-2.764-3.86a1.006 1.006 0 00-1.236.979V26l-5 6"
-                  />
-                  <path
-                    className="stroke-current text-white-main"
-                    d="M22 30h4v12h-4z"
-                  />
-                </g>
-              </svg>
-              <h4 className="h4 mb-2">Batteries Included</h4>
-              <p className="text-center text-lg text-gray-600">
-                Pear comes fully setup with shortcuts and terminal plugins used
-                to supercharge development productivity.
-              </p>
-            </div>
+function SingleFeature(props) {
```

Comment:
Since we are using Typescript we should infer all types. so at the top we can define it like this:
```ts
interface FeatureInfo {
  header: string;
  description: string;
  imageWidth: number;
  imageSource: string;
}
```
```ts
interface SingleFeatureProps {
  info: FeatureInfo;
}
```

and then back to the function we can do this:
```ts
function SingleFeature(props: SingleFeatureProps)
```